<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-22T00:49:44 IST" name="Default suite" finished-at="2022-02-22T00:50:37 IST" duration-ms="53739">
    <groups>
    </groups>
    <test started-at="2022-02-22T00:49:44 IST" name="Default test" finished-at="2022-02-22T00:50:37 IST" duration-ms="53739">
      <class name="verifyProfileTest">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:verifyProfileTest@72f926e6]" started-at="2022-02-22T00:49:44 IST" name="openBrowser" finished-at="2022-02-22T00:50:07 IST" duration-ms="23158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="logIntoZerodha()[pri:0, instance:verifyProfileTest@72f926e6]" started-at="2022-02-22T00:50:07 IST" name="logIntoZerodha" finished-at="2022-02-22T00:50:29 IST" duration-ms="21587" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIntoZerodha -->
        <test-method signature="VerifyUserName()[pri:0, instance:verifyProfileTest@72f926e6]" started-at="2022-02-22T00:50:29 IST" name="VerifyUserName" finished-at="2022-02-22T00:50:34 IST" duration-ms="5242" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Raya] but found [JT2222]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Raya] but found [JT2222]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at verifyProfileTest.VerifyUserName(verifyProfileTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyUserName -->
        <test-method is-config="true" signature="result(org.testng.ITestResult)[pri:0, instance:verifyProfileTest@72f926e6]" started-at="2022-02-22T00:50:34 IST" name="result" finished-at="2022-02-22T00:50:37 IST" duration-ms="3176" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=VerifyUserName status=FAILURE method=verifyProfileTest.VerifyUserName()[pri:0, instance:verifyProfileTest@72f926e6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- result -->
      </class> <!-- verifyProfileTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
